static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -422,6 +423,8 @@ const AddViewButton = styled(Button)`
   font-weight: normal;
   padding: ${space(0.5)};
   margin-right: ${space(0.5)};
+  border: none;
+  bottom: -1px;
```

Comment:
the temp tab was 1 pixel over the bottom border of the tab bar previously:

<img width="273" alt="Screenshot 2024-09-16 at 3 12 34 PM" src="https://github.com/user-attachments/assets/949fb4a2-3f97-429e-80e4-f720e75d2524">